internal/cmd/gocdk/internal/templates/init.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package templates contains string literals for use by the gocdk
+// command when creating user project files.
+package templates
+
+var InitTemplates = map[string]string{
```

Comment:
Travis wants you to add a declaration for this new package. Since we're going to delete it soon as part of adding them to `_static`, I suggest just deleting this file and moving its contents into `init.go`.